src/coreComponents/constitutive/fluid/multifluid/CO2Brine/functions/BrineEnthalpy.cpp

```diff
@@ -73,21 +73,34 @@ real64 michaelidesBrineEnthalpy( real64 const & T,
 
 void calculateBrineEnthalpy( PTTableCoordinates const & tableCoords,
                              real64 const & m,
-                             array1d< real64 > const & enthalpies )
+                             array1d< real64 > const & enthalpies,
+                             bool const printTable )
```

Comment:
Instead of interleaving table calculations with output, would it be feasible to have a single function
```
TableFunction::print( std::ostream & os, std::vector<string> const & columnNames )
```
that is called _after_ the table is populated?

Reply:
considered that but honestly i have troubles dealing with table format and how values are stored there...

Reply:
ok, should be better in the latest version, thanks!